zulipterminal/ui_tools/views.py

```diff
@@ -1525,13 +1525,13 @@ def __init__(
         controller: Any,
         msg: Message,
         title: str,
-        topic_links: "OrderedDict[str, Tuple[str, int, bool]]",
         message_links: "OrderedDict[str, Tuple[str, int, bool]]",
+        topic_links: "OrderedDict[str, Tuple[str, int, bool]]",
```

Comment:
This change isn't strictly necessary, and right now will cause an issue since you've not change the call sites.